### PR TITLE
Fix artik audio bootup

### PR DESCRIPTION
**Pull request by btheosam**

### CODE REVIEW

File: os/arch/arm/src/artik053/src/artik053_alc5658.c [hunk 1]
```diff
@@ -0,0 +1,341 @@
+/****************************************************************************
+ *
+ * Copyright 2016 Samsung Electronics All Rights Reserved.
```
Comment (sunghan-chang) [inline os/arch/arm/src/artik053/src/artik053_alc5658.c]: 2016 -> 2017

File: os/arch/arm/src/artik053/src/artik053_alc5658.c [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+		if (!i2c) {
+			auddbg("ERROR: Failed to initialize CODEC I2C%d\n");
+			ret = -ENODEV;
+			goto errout;
+		}
+
+		/* Get an instance of the I2S interface for the ALC5658 data channel */
+
+		i2s = s5j_i2s_initialize();
+		if (!i2s) {
```
Comment (shivgarg) [inline os/arch/arm/src/artik053/src/artik053_alc5658.c]: It should be called with 0 as argument, 0 indicating the the i2s port.

File: os/arch/arm/src/artik053/src/artik053_boot.c [hunk 3]
```diff
@@ -126,14 +126,29 @@ static void board_gpio_initialize(void)
  ****************************************************************************/
 static void board_i2c_initialize(void)
 {
-#ifdef CONFIG_I2C
-#ifdef CONFIG_S5J_I2C
+#if defined  CONFIG_I2C && defined CONFIG_S5J_I2C
```
[the remark this answers is not in the document]
Comment (btheosam) [reply os/arch/arm/src/artik053/src/artik053_boot.c]: Fixed.

File: os/arch/arm/src/artik053/src/artik053_alc5658.c (hunk 1 above)
Comment (btheosam) [reply os/arch/arm/src/artik053/src/artik053_alc5658.c]: Fixed

File: os/arch/arm/src/artik053/src/artik053_alc5658.c (hunk 2 above)
Comment (btheosam) [reply os/arch/arm/src/artik053/src/artik053_alc5658.c]: Fixed.

**Pull request merged**